src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -15121,7 +15121,7 @@ public Child2(int k)
 
             compilation.VerifyDiagnostics(expected);
 
-            compilation.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, compilation.SyntaxTrees.Single(), null, true).Verify(expected);
+            compilation.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, compilation.SyntaxTrees.Single(), null, true, false, CancellationToken.None).Verify(expected);
```

Comment:
Please use named args.

minor: You could avoid any changes to this file by making the last two (or three) arguments optional as I suggested above.